package.json

```diff
@@ -67,7 +67,7 @@
     "numeral": "^2.0.6",
     "pace-progress": "git+https://github.com/getredash/pace.git",
     "pivottable": "^2.15.0",
-    "plotly.js": "1.30.1",
+    "plotly.js": "^1.41.3",
```

Comment:
Let's have a specific version here (`"1.41.3"`) as in the past there were cases of regressions in their releases, so we don't want random versions getting picked up in our builds.